crates/turborepo-lib/src/run/summary/spaces.rs

```diff
@@ -148,9 +148,16 @@ impl SpacesClient {
         let space_id = space_id?;
         let is_linked = api_auth.as_ref().map_or(false, |auth| auth.is_linked());
         if !is_linked {
+            let base = api_client.base_url();
+            let login_command = if base.contains("vercel") {
+                "turbo login".to_string()
+            } else {
+                format!("turbo login --api {}", base)
+            };
```

Comment:
I believe this should happen in `CommandBase::api_auth` as there we have more information such as what APIs have logins associated with them.

This warning will also only show if spaces are enabled which most projects (at the moment) do not have toggled on.

Reply:
Fair point, this was where all my debugging led me. I'll switch up the wording in the `api_auth` but keep the same concept.

Reply:
Moving this instead to the run where it'll only run on a linked project.